src/app/components/newproposal/FundingInput.tsx

```diff
@@ -7,29 +7,26 @@ import AddressInputPanel from 'app/components/AddressInputPanel';
 import CurrencyInputPanel from 'app/components/CurrencyInputPanel';
 import { BoxPanel } from 'app/components/newproposal/RatioInput';
 import { PROPOSAL_CONFIG, CURRENCY_LIST } from 'app/containers/NewProposalPage/constant';
+import { CurrencyAmount } from 'types';
 
 export interface CurrencyValue {
   recipient: string;
-  amounts: Amount;
+  amounts: Amounts;
```

Comment:
Please remove the `Amounts` data structure. and use the `CurrencyAmount` data structure.